src/node_file.cc

```diff
@@ -775,7 +775,7 @@ void Access(const FunctionCallbackInfo<Value>& args) {
               uv_fs_access, *path, mode);
   } else {  // access(path, mode, undefined, ctx)
     CHECK_EQ(argc, 4);
-    fs_req_wrap req_wrap;
+    FSReqWrapSync req_wrap;
```

Comment:
In retrospect variables like this can be renamed `req_wrap_sync` and those `FSReqBase* ` can be renamed `req_wrap_async` to avoid the confusion of name shadowing..not really what this PR needs to deal with though.

Reply:
I noticed that too and think we should change that. I'll follow up this is in a different PR. Thanks!